plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/contentmodel/ContentTypeResourceController.java

```diff
@@ -0,0 +1,232 @@
+package org.entando.entando.plugins.jacms.web.contentmodel;
+
+import com.agiletec.aps.system.services.role.Permission;
+import com.agiletec.plugins.jacms.aps.system.services.contentmodel.model.*;
+import com.google.common.collect.ImmutableMap;
+import org.apache.commons.lang3.StringUtils;
+import org.entando.entando.aps.system.services.entity.model.*;
+import org.entando.entando.plugins.jacms.aps.system.services.ContentTypeService;
+import org.entando.entando.plugins.jacms.web.contentmodel.validator.ContentTypeValidator;
+import org.entando.entando.web.common.annotation.RestAccessControl;
+import org.entando.entando.web.common.exceptions.*;
+import org.entando.entando.web.common.model.*;
+import org.entando.entando.web.userprofile.validator.ProfileTypeValidator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.net.*;
+import java.util.*;
+
+@RestController
+public class ContentTypeResourceController implements ContentTypeResource {
+
+    private static final String CONTENT_TYPE_CODE = "contentTypeCode";
+    private static final String ATTRIBUTE_CODE = "attributeCode";
+    private static final String MOVEMENT = "movement";
+
+    private final ContentTypeService service;
+    private final ContentTypeValidator validator;
+
+    @Autowired
+    public ContentTypeResourceController(ContentTypeService service, ContentTypeValidator validator) {
+        this.service = service;
+        this.validator = validator;
+    }
+
+    @Override
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    public ResponseEntity<ContentTypeDto> create(
```

Comment:
I think that our API should always be in the format `{payload:{}, metadata: {}, errors:[]}`, so in this case the method return type should be `ResponseEntity<SimpleRestResponse<ContentTypeDto>>`, but please check this with @joewhite101.

Reply:
I'll change them all to SimpleRestResponse to make everything consistent, that will be easier for UI handling later

Reply:
Fixed